server/data/quotes.json

```diff
@@ -261,5 +261,17 @@
  {
   "sentence": "We die. That may be the meaning of life. But we do language. That may be the measure of our lives.",
   "author": "Toni Morrison"
+ },
+ {
+  "sentence": "You can’t get much done in life if you only work on the days when you feel good.",
```

Comment:
Hi there @arpitBhalla! Sorry just looked at the quotes, but they should be related to learning a foreign language :)